node/src/database/db_migrations/migrations/migration_7_to_8.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+
+use crate::database::db_migrations::db_migrator::DatabaseMigration;
+use crate::database::db_migrations::migrator_utils::DBMigDeclarator;
+
+#[allow(non_camel_case_types)]
+pub struct Migrate_7_to_8;
+
+impl DatabaseMigration for Migrate_7_to_8 {
+    fn migrate<'a>(
+        &self,
+        mig_declaration_utilities: Box<dyn DBMigDeclarator + 'a>,
+    ) -> rusqlite::Result<()> {
+        let statement =
+            "INSERT INTO config (name, value, encrypted) VALUES ('min_hops_count', '3', 0)";
+        mig_declaration_utilities.execute_upon_transaction(&[&statement])
```

Comment:
Maybe the second `&` in the nested slices can be omitted? Thinking that statement is also a slice `&str`.

Later when I've seen the test I had to go back here and see it again. My late impression is that the number 3 should actually not be exactly hard-coded but it should be the DEFAULT_MIN_HOPS_COUNT. So my comment from the beginning might become irrelevant because you will need `format!()`.

Reply:
Done 👍